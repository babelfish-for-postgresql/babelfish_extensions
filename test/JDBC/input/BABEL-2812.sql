declare @refreshTime varchar(20) = '17:30:00'
declare @yyyymmdd    varchar(20) = '20211212'
select CONVERT(datetime,@refreshTime,14) + @yyyymmdd as NextTime
go

declare @refreshTime datetime = '17:30:00'
declare @yyyymmdd    datetime = '20211212'
select  @refreshTime + @yyyymmdd as NextTime
go

declare @refreshTime datetime = '17:30:00'
declare @yyyymmdd    datetime = '20211212'
SELECT @yyyymmdd - @refreshTime
go


declare @refreshTime datetime = '17:30:00'
declare @yyyymmdd    datetime = '20211212'
SELECT DATEADD(day ,DATEDIFF(day, 0, @yyyymmdd) ,@refreshTime) as NextTime;
go
