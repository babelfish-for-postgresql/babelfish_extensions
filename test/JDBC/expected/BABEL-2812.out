declare @refreshTime varchar(20) = '17:30:00'
declare @yyyymmdd    varchar(20) = '20211212'
select CONVERT(datetime,@refreshTime,14) + @yyyymmdd as NextTime
go
~~START~~
datetime
2021-12-12 17:30:00.0
~~END~~


declare @refreshTime datetime = '17:30:00'
declare @yyyymmdd    datetime = '20211212'
select  @refreshTime + @yyyymmdd as NextTime
go
~~START~~
datetime
2021-12-12 17:30:00.0
~~END~~


declare @refreshTime datetime = '17:30:00'
declare @yyyymmdd    datetime = '20211212'
SELECT @yyyymmdd - @refreshTime
go
~~START~~
datetime
2021-12-11 06:30:00.0
~~END~~



declare @refreshTime datetime = '17:30:00'
declare @yyyymmdd    datetime = '20211212'
SELECT DATEADD(day ,DATEDIFF(day, 0, @yyyymmdd) ,@refreshTime) as NextTime;
go
~~START~~
datetime
2021-12-12 17:30:00.0
~~END~~

